### PR TITLE
pool: Adding TokenData helper.

### PR DESCRIPTION
**Pull request by mijovic**

### CODE REVIEW

File: contracts/TempusPool.sol [hunk 1]
```diff
@@ -11,6 +11,12 @@ import "./token/YieldShare.sol";
 import "./math/Fixed256xVar.sol";
 import "./utils/UntrustedERC20.sol";
 
+/// @dev helper struct to store name and symbol for the token
+struct TokenData {
```
Comment (mijovic) [inline contracts/TempusPool.sol:15]: Not sure about name for this struct

File: contracts/mocks/aave/AavePoolMock.sol [hunk 2]
```diff
@@ -24,11 +24,11 @@ contract AavePoolMock is ILendingPool {
         IERC20 asset,
         uint128 index,
         uint8 decimals,
-        string memory aTokenName,
+        string memory aTokenData,
```
Comment (mijovic) [inline contracts/mocks/aave/AavePoolMock.sol:27]: find and replace issue, will revert it now

File: contracts/TempusPool.sol [hunk 3]
```diff
@@ -58,10 +64,8 @@ abstract contract TempusPool is ITempusPool {
     /// @param initInterestRate initial interest rate of the pool
     /// @param exchangeRateOne 1.0 expressed in exchange rate decimal precision
     /// @param estimatedFinalYield estimated yield for the whole lifetime of the pool
-    /// @param principalName name of Tempus Principal Share
-    /// @param principalSymbol symbol of Tempus Principal Share
-    /// @param yieldName name of Tempus Yield Share
-    /// @param yieldSymbol symbol of Tempus Yield Share
+    /// @param principalsData Tempus Principals name and symbol
+    /// @param yieldsData name of Tempus Yields name and symbol
```
Comment (axic) [inline contracts/TempusPool.sol:68]:
```suggestion
    /// @param yieldsData Tempus Yields name and symbol
```

File: contracts/TempusPool.sol (hunk 1 above)
Comment (axic) [reply contracts/TempusPool.sol:15]: What is the goal of this change?

Comment (mijovic) [reply contracts/TempusPool.sol:15]: Adding additional param raises stack to deep error, so I did this in separate PR to have 1 change at a time

Comment (axic) [reply contracts/TempusPool.sol:15]: Where does it raise stack too deep error?

Comment (axic) [reply contracts/TempusPool.sol:15]: Also perhaps `TokenSettings` / `TokenProperties` may be a better name?

**Review by axic — APPROVED** [review]
Otherwise looks good.

File: contracts/TempusPool.sol (hunk 1 above)
Comment (axic) [reply contracts/TempusPool.sol:15]: > Where does it raise stack too deep error?

Ah it raises in #422.

**Pull request merged**